### PR TITLE
update sriov network template to define spoofChk/trust/ipam

### PR DESCRIPTION
**Pull request by manurodriguez**
build-depends: https://github.com/dci-labs/inventories/pull/152

### CODE REVIEW

**Review by tonyskapunk — APPROVED** [review]
LGTM

**Comment by manurodriguez** [comment]
In the last test the Trex tests ran successfully but the TNF test suite failed to start, maybe because this is an OCP 4.7 cluster, I'll test later with 4.12

File: testpmd/hooks/templates/sriov-network.yml.j2 [hunk 1]
```diff
@@ -7,6 +7,6 @@ spec:
   networkNamespace: "{{ cnf_namespace }}"
   vlan: {{ sriov['vlan']|int }}
   resourceName: "{{ sriov['resource'] }}"
-  spoofChk: "on"
-  trust: "on"
-  ipam: '{ }'
+  spoofChk: "{{ sriov['spoofchk'] | default('off') }}"
```
Comment (capolrik) [inline testpmd/hooks/templates/sriov-network.yml.j2:10]: From I understood of the last sync, spoofChk=off and trust=on is for DPDK workload but we want the 'opposite' for the blueprint spoofChk=on and trust=off, isn't it?

**Review by capolrik — APPROVED** [review]

**Comment by ramperher** [comment]
> In the last test the Trex tests ran successfully but the TNF test suite failed to start, maybe because this is an OCP 4.7 cluster, I'll test later with 4.12

Yeah, tnf is not compliant anymore with OCP 4.7, that's why we don't launch example-cnf in that version on dailies.

**Review by ramperher — APPROVED** [review]

File: testpmd/hooks/templates/sriov-network.yml.j2 (hunk 1 above)
Comment (manurodriguez) [reply testpmd/hooks/templates/sriov-network.yml.j2:10]: I'll ask Franck, my understanding was that we wanted `spoofChk=off` with our example-cnf , but I'll check, it wasn't clear to me if we were going to test both DPDK and Kernel workloads,

Comment (manurodriguez) [reply testpmd/hooks/templates/sriov-network.yml.j2:10]: I clarified this point, we want two workloads:
- one with DPDK (example-cnf) using `spoofChk=off and trust=on`
- another workload for Kernel using `spoofChk=on and trust=off` (we'll deploy some statefulapp hook or something else)

**Comment by manurodriguez** [comment]
check dallas ocp-4.12-vanilla example-cnf-blue

**Review by tonyskapunk — APPROVED** [review]
LGTM

**Pull request merged**